ext/telemetry/errors.js

```diff
@@ -1,21 +1,18 @@
 'use strict';
 
+var defaultHandler = defaultHandler || window.onerror;
```

Comment:
I think that `defaultOnError` should be set only when entering `trackUncaught` to store the one that is actually used. I'm wary of code like that that run on the module loading. We don't control which module is loaded first so a user could have installed another package that monitor uncaught exceptions (I think there's a few on atom.io) and if that package set a  different `onerror` handler after our own module was loaded will screw up the user setup by restoring a different handler.